docs/goals.md

```diff
@@ -12,16 +12,14 @@ of some data value or values.
 
 The design goals are listed below.
 
-1.  Express grammatical features, such as plurals, genders, and inflections.
-
-2.  Express other variance in translation, due to linguistic and regional
-    features, the presentation media, context, circumstance, and other factors.
+1. Allow users to write messages that are both grammatically
+   correct and can be translated in a grammatically correct manner
+   to languages that have different requirements. For example, plurals.
```

Comment:
Accepting just "plurals" as a sufficient example seems to expect a lot of the reader. Should this sentence fragment be expanded?

Reply:
I'm ok with this as a goal. I have no doubt that we make huge progress towards this with MF2.0. (I can relay caveats elsewhere.)